docs/development/issue_labeling.md

```diff
@@ -0,0 +1,221 @@
+
+Add a label `needs reproduction` if the issue can only be fixed once there's a reproduction.
+Add the label `reproduced` once there is a public reproduction.
+
+QUESTION: When/who should use the `help wanted` label?
+
+### Miscellaneous
+
+<details>
+    <summary>Miscellaneous</summary>
+
+    gitfs
+    hacktoberfest
+    hacktoberfest-accepted
+    internal
+    ready
+    self hosted
+
+</details>
+
+QUESTION: What is the `gitfs` label about?
+QUESTION: Should we document the `hacktoberfest` labels?
+QUESTION: What is the `internal` label about?
+QUESTION: Do we even use the `ready` label now Renovate has a project board?
+QUESTION: I think `self hosted` belongs in the `platform:` type label section instead???
+
+### Datasource
+
+<details>
+    <summary>Datasource</summary>
+
+    datasource:docker
+    datasource:git-submodule
+    datasource:git-labels
+    datasource:jenkins
+    datasource:maven
+    datasource:nuget
+    datasource:packagist
+    datasource:pypi
+    datasource:rubygems
+    datasource:terraform-module
+    datasource:terraform-provider
+
+</details>
+
+QUESTION: When should somebody use a datasource label?
+
+### Manager
+
+<details>
+    <summary>Manager</summary>
+
+    manager:bazel
+    manager:buildkite
+    manager:bundler
+    manager:cargo
+    manager:circleci
+    manager:cocoapods
+    manager:composer
+    manager:docker-compose
+    manager:dockerfile
+    manager:github-actions
+    manager:gitlab-ci
+    manager:gomod
+    manager:gradle
+    manager:helm
+    manager:helm-values
+    manager:kubernetes
+    manager:kustomize
+    manager:maven
+    manager:meteor
+    manager:mix
+    manager:npm
+    manager:nuget
+    manager:pip_requirements
+    manager:pip_setup
+    manager:pipenv
+    manager:poetry
+    manager:ruby-version
+    manager:sbt
+    manager:swift
+    manager:terraform
+    manager:travis
+
+</details>
+
+"manager" is short for "package manager".
+Add the relevant manager labels to the issue.
+If there are multiple managers affected, add labels for all of them.
+
+### New stuff
+
+<details>
+    <summary>New stuff</summary>
+
+    new datasource
+    new package manager
+    new platform
+    new versioning
+
+</details>
+
+Apply these labels when somebody opens a `feature` type issue requesting a new datasource, package manager, platform, or new versioning scheme.
+
+### Platform
+
+<details>
+    <summary>Platform labels</summary>
+
+    platform:azure
+    platform:bitbucket
+    platform:gitea
+    platform:github
+    platform:gitlab
+
+</details>
+
+Use these to mark the platform that is affected by this issue.
+
+QUESTION: Should the `self hosted` label be in this category as well? Or are we missing a `platform:self-hosted` label?
+
+### Worker
+
+<details>
+    <summary>Worker</summary>
+
+    worker:branch
+    worker:global
+    worker:onboarding
+    worker:pr
+
+</details>
+
+QUESTION: I have no clue what a worker even is....
```

Comment:
Worker is kind of the "core" logic of Renovate. I think it's a good idea to have these, or they haven't been used heavily so far. In theory any feature or bug should apply to either a platform, manager, datasource, versioning or a worker part.